src/Features/CSharp/Portable/ConvertToRawString/ConvertRegularStringToRawStringCodeRefactoringProvider.cs

```diff
@@ -3,111 +3,44 @@
 // See the LICENSE file in the project root for more information.
```

Comment:
i kep the name of this file the same to keep the diff clear.    the CodeREfactoringProvider code was in here, as well as the code to deal with normal (non-interpolated) strings.

Now, this file just contains the code to deal with the strings.  Everything related to the CodeRefactoringProvider moved into a separate file.  The CodeRefactoringProvider now calls into the helpers for normal strings and interpolated strings to do the actual processing.